src/ui/page/search/index.html

```diff
@@ -21,7 +21,7 @@ <h1 class="heading-xlarge heading heading--page-heading ">
                 {{/if}} {{#if searchResults && query}}
                 <p class="lede">There were
                     <strong class="lpg-search-amount">{{#if searchResults.totalResults}} {{searchResults.totalResults}} {{else }} no {{/if}}</strong>
-                    results for
+                    {{currentReq.__n('%s result', searchResults.totalResults)}} for
```

Comment:
Does this print out right? Looks to me (without running it) it'd print "There were 1 1 result for ..."